scripts/build_test.py

```
@@ -1008,27 +590,19 @@ def test_build_with_prod_env(self) -> None:
         clean_swap = self.swap_with_checks(
             build, 'clean', lambda: None, expected_args=[()]
         )
-        inject_angular_css_hashes_swap = self.swap_with_checks(
-            build, 'inject_angular_css_hashes', lambda: None, expected_args=[()]
+        sync_angular_css_hashes_swap = self.swap_with_checks(
+            build, 'sync_angular_css_hashes', lambda: None, expected_args=[()]
         )
 
-        with ensure_files_exist_swap, build_using_webpack_swap, clean_swap:
+        with ensure_files_exist_swap, clean_swap:
             with modify_constants_swap, build_using_ng_swap:
```

Ruff (0.15.15)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)